doc/sphinx/addendum/type-classes.rst

```diff
@@ -452,6 +452,11 @@ Summary of the commands
    + Use the :cmd:`Typeclasses eauto` command to customize the behavior of
      this tactic.
 
+   :n:`{| bfs | dfs }
+     Sets the search strategy to depth-first
+     search (the default) or breadth-first search. The search strategy
+     can also be set with :flag:`Typeclasses Iterative Deepening`.
```

Comment:
```suggestion
     Specifies whether to use breadth-first search or depth-first search.  The default is depth-first search,
     which can be changed with the :flag:`Typeclasses Iterative Deepening` flag.
```

"Sets" suggests that a change the persists after the tactic.
Isn't the default determined by the `Typeclasses Iterative Deepening`?
If that's correct.

And, if so, perhaps you would also tweak the wording for `Typeclasses eauto` (change the label to ```:n:`{| bfs | dfs }` ``` and swap the underlined wording so the order of the descriptions matches the order in the label

![image](https://user-images.githubusercontent.com/1253341/126555704-aafa2d8e-3c5d-4075-a99d-e5d61f3db706.png)

Reply:
Yes, I copied this from further down, I'll change both.